src/handler_tunnel_chain.js

```diff
@@ -20,12 +20,16 @@ export default class HandlerTunnelChain extends HandlerBase {
     run() {
         this.log('Connecting to upstream proxy...');
 
+        const targetHost = `${this.trgParsed.hostname}:${this.trgParsed.port}`;
+
         const options = {
             method: 'CONNECT',
             hostname: this.upstreamProxyUrlParsed.hostname,
             port: this.upstreamProxyUrlParsed.port,
-            path: `${this.trgParsed.hostname}:${this.trgParsed.port}`,
-            headers: {},
+            path: targetHost,
+            headers: {
+                host: targetHost,
```

Comment:
This should be `Host`

Reply:
I will change though header names are case insensitive so that doesn't really matter

Reply:
Great, thanks. You're right, but it's typical to send the headers with the right case.